### DISCUSSION TITLE
Copilot X Chat Icon doesn't appear in the activity bar

### DISCUSSION THREAD

**Discussion by tjprescott** [community/community#53902] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I followed the directions to install the Github Copilot X dogfood build.
- installed VSCode Insiders build
- installed the Github Copilot Nightly Build
- Restarted VSCode
- Logged in to Github Copilot

However, I don't have the chat icon in the activity bar like I'm supposed to, and it seems to operate just like regular copilot. Did I miss a crucial step?

![image](https://user-images.githubusercontent.com/5723682/234961317-d3aa3faf-168a-467c-9bd7-f058642bff69.png)

**Comment by tjprescott** [comment]
Well, despite searching for existing discussions and finding none, as soon as I posted this I saw a lot of others had posted this same thing!

I installed this as prompted by Doug's email to install and use the dogfood builds. It says I am on the waitlist for the Github Copilot X technical preview.

Is there anything else I need to do?

**Reply by tjprescott** [reply, marked as answer]
As an FTE, I had to do a few extra steps to enable this. This link was very helpful:
https://eng.ms/docs/more/github-inside-microsoft/troubleshoot/copilot

**Reply by dillweed** [reply]
I get an "Invalid Tenant" error when loading that page. 

<img width="591" alt="Screenshot 2023-05-02 at 10 10 47 AM" src="https://user-images.githubusercontent.com/4053467/235736594-d0ae7ae7-eeb9-406a-b291-3ca4bd899181.png">

I'm having the same issue with a missing Copilot chat icon on the activity bar. Can you describe the steps you took to solve it?

**Reply by tjprescott** [reply]
I needed to link my GH account to an enterprise MS account (which I thought I had already done--been here 7 years using GH!)
https://repos.opensource.microsoft.com/link

**Comment by rfink** [comment]
Same issue.
